src/lib/datasource/FileReadWriteDatasource.js

```diff
@@ -55,6 +58,47 @@ class FileReadWriteDatasource extends FileReadDatasource {
         });
     }
 
+    /**
+     * 
+     * @param {*} keysArray 
+     */
+    mget(keysArray) {
```

Comment:
This seems to be a copy-paste of same method in parent class

Reply:
done